src/models.rs

```diff
@@ -0,0 +1,38 @@
+use near_sdk::serde::{Serialize, Deserialize};
+
+use crate::*;
+
+/// Information about a specific public key. Should be returned in the `get_key_information` view method.
+/// Part of the linkdrop NEP
+#[derive(Serialize)]
+#[serde(crate = "near_sdk::serde")]
+pub struct KeyInfo {
+    /// yoctoNEAR$ amount that will be sent to the claiming account (either new or existing)
+    /// when the key is successfully used.
+    pub balance: U128,
+}
+
+
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "near_sdk::serde")]
+/// Information about any limited access keys that are being added to the account as part of `create_account_advanced`.
+pub struct LimitedAccessKey {
+    /// The public key of the limited access key.
+    pub public_key: PublicKey,
+    /// The amount of yoctoNEAR$ that can be spent on Gas by this key.
+    pub allowance: U128,
+    /// Which contract should this key be allowed to call.
+    pub receiver_id: AccountId,
+    /// Which methods should this key be allowed to call.
+    pub method_names: String,
+}
+    
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "near_sdk::serde")]
+/// Options for `create_account_advanced`.
+pub struct CreateAccountOptions {
+    pub full_access_keys: Option<Vec<PublicKey>>,
+    pub limited_access_keys: Option<Vec<LimitedAccessKey>>,
```

Comment:
probably want to combine these to make a `AccessKey` like the one found in https://docs.rs/near-primitives/0.15.0/near_primitives/account/struct.AccessKey.html. Then you just have `access_keys: Option<Vec<AccessKey>>` at the end of the day to simplify the code

Reply:
In talking with Daryl and rest of team, we decided it would be best to go with this interface for ease of development (spec found here: https://docs.google.com/document/d/18v9UCzYkFCvsBJuq3_XSevqw2B8o0geiJyWgdH-z0fw/edit#)

Reply:
I don't see anything relating to this in that doc, or probably you guys discussed offline. Like effectively, the code change would be the same as these two separate fields. Really wouldn't matter which you use since they're functionally equivalent unless you have a huge amount of keys and don't want to filter for full access over limited